backend/automated-tests/src/main/kotlin/br/com/zup/beagle/automatedtests/builders/ButtonScreenBuilder.kt

```diff
@@ -58,49 +58,57 @@ object ButtonScreenBuilder {
-                            margin = EdgeValue(
-                                top = 15.unitReal()
-                            ),
-                            flex = Flex(alignContent = alignContent)
-                        )
-                    ),
+            child = createContainer(alignContent, marginTop)
+        )
+    }
 
-                    createButton(
-                        text = "Button with style",
-                        styleId = "DesignSystem.Button.ScreenButton",
-                        style = Style(
-                            margin = EdgeValue(
-                                top = 15.unitReal()
-                            )
-                        )
+    private fun createContainer(alignContent: AlignContent, marginTop: UnitValue) = Container(
+        children = listOf(
+            createButton(
+                text = "Button",
+                style = Style(
+                    margin = EdgeValue(
+                        top = 15.unitReal()
                     ),
+                    flex = Flex(alignContent = alignContent)
+                )
+            ),
 
-                    buttonWithAppearanceAndStyle(text = "Button with Appearance").applyStyle(
-                        style = Style(
-                            margin = EdgeValue(
-                                top = 15.unitReal()
-                            )
-                        )
-                    ),
+            createButton(
+                text = "Button with style",
+                styleId = "DesignSystem.Button.ScreenButton",
+                style = Style(
+                    margin = EdgeValue(
+                        top = 15.unitReal()
+                    )
+                )
+            ),
 
-                    buttonWithAppearanceAndStyle(
-                        text = "Button with Appearance and style",
-                        styleId = BUTTON_STYLE_APPEARANCE
-                    ).applyStyle(
-                        Style(
-                            margin = EdgeValue(
-                                top = marginTop
-                            ))
+            buttonWithAppearanceAndStyle(text = "Button with Appearance").applyStyle(
+                style = Style(
+                    margin = EdgeValue(
+                        top = 15.unitReal()
                     )
                 )
+            ),
+
+            buttonWithAppearanceAndStyle(
+                text = "Button with Appearance and style",
+                styleId = BUTTON_STYLE_APPEARANCE
+            ).applyStyle(
+                Style(
+                    margin = EdgeValue(
+                        top = marginTop
+                    ))
+            ),
+            Button(
+                text = "Disabled Button",
+                styleId = "DesignSystem.Button.ScreenButton",
+                onPress = listOf(Alert(message = "This button must be disabled")),
+                disabled = true
```

Comment:
What do you think of using this property from context to reactively enable and disable?
```suggestion
                disabled = expressionOf("@{context.disabled}")
```

Reply:
done